cli/crawl-cli.js

```diff
@@ -233,6 +273,16 @@ if (!urls || !program.output) {
         return `http://${url}`;
     });
 
+    if (fs.existsSync(`${program.output}/screenshots`) && !forceOverwrite) {
+        // eslint-disable-next-line no-console
+        console.log(chalk.red('Screenshot folder already exists'), 'Use -f to overwrite.');
+    } else if (program.screenshotLogging) {
+        screenshotHelper.loadScreenshotList(program.screenshotLogging);
+        if (!fs.existsSync(`${program.output}/screenshots`)) {
+            fs.mkdirSync(`${program.output}/screenshots`);
+        }
+    }
```

Comment:
This block should happen after the output dir is created, otherwise we try to make the screenshots dir in it before it exists.